jest.setup.js

```diff
@@ -0,0 +1 @@
+jest.setTimeout(10000) // in milliseconds
```

Comment:
I kept having the `fetches wrapper jars checksums` test fail for me because it took upwards of 5-6 seconds to download all of the checksums